### ISSUE TITLE
Was the Lseg trained in an inductive zero-shot setting?

### ISSUE THREAD

**Issue by Genie-Kim** [isl-org/lang-seg#26]
Thanks for your excellent research.

I have a question about Lseg's zero-shot setting.
Was the Lseg trained in an inductive zero-shot setting?

What is the difference between inductive zero-shot setting and language-driven semantic segmentation setting in the training step?

The settings for zero-shot semantic segmentation are confusing. Please help me.

Thank you for reading.

**Comment by Boyiliee** [comment]
Hi, @Genie-Kim ,

Thank you for your interest in LSeg!

As has been mentioned in the paper, we follow the experimental setting as HSNet, however, the difference is that LSeg has no chance to look at any samples of the test set. We didn't follow some zero-shot semantic segmentation settings, since there are some unreasonable design in the setting, and has been less used than the HSNet setting. Therefore we use the latter.

Hope this helps.
